sklearn/datasets/_base.py

```diff
@@ -578,6 +578,9 @@ def load_iris(*, return_X_y=False, as_frame=False):
             .. versionadded:: 0.20
 
     (data, target) : tuple if ``return_X_y`` is True
+        Returns a tuple of two ndarray of shape (n_samples, n_features)
```

Comment:
```suggestion
        A tuple of two ndarray of shape (n_samples, n_features)
```

Reply:
To complete @ogrisel's comment https://github.com/scikit-learn/scikit-learn/pull/21760/files#r755364536.